src/components/PaginationFooter/Dropdowns/PageOptions.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+interface PageOptionsProps {
+  totalPages: number;
+  page: number;
+}
+
+export const MAX_PAGE_OPTIONS_LENGTH = 1000;
+
+export const HALF_MAX_LENGTH = MAX_PAGE_OPTIONS_LENGTH / 2;
+
+export const ELLIPSIS = '…';
```

Comment:
maybe these should be placed in constants dir? 
and if it grows, we know that it will not hurts us because more or less most of it's values is being used during each render

Reply:
If would only put them in a "shared" constant dir or file if they are used imported by other modules. Currently they are not. They are exported for testing purposes so I don't really see any value in moving them.

I can see a value in exposing `ELLIPSIS` as a const to both `sn-table` and `sn-pivot-table` repos for use there. Since they also have that character defined. But I'd say that is for another PR.

Reply:
ok then that would means that we would need a small refactor pr to move `HEAD_CELL_MENU...` prefixed constants into a local constant file for HeadCellMenu, will take care of it in future